src/types.ts

```diff
@@ -29,16 +24,25 @@ export interface AutoIncrementOptionsID {
   field?: string;
   /**
    * Set the tracker model name
-   * @default idtracker
+   * @default identitycounter
    */
   trackerModelName?: string;
+  /**
+   * model to configure the plugin for
+   */
+  model: string;
+  /**
+   * the count should start at
+   * @default 0
+   */
+  startAt?: number;
 }
 
 export interface AutoIncrementIDTrackerSpec {
   /** The ModelName from the current model */
-  m: string;
+  model: string;
```

Comment:
Why change that to the full name?
-> it was smaller so that it requires less bandwidth and lower storage footprint

(Note: deleted original, because it was in the wrong spot & and was an single comment, but should be in this review)

Reply:
it should be readable. that's why changed to full name.

Reply:
its fine by me, but just thought that it should require as low as possible
-> maybe (when asked to) implement an model that makes it "short"?